src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -123,7 +124,30 @@ function repeatableInputToObj(container_name) {
          */
         function bpFieldInitRepeatableElement(element) {
 
-            var field_name = element.attr('name');
+            let field_name = element.attr('name');
+            let form = element.closest('form');
+            let form_events = jQuery._data(form[0], 'events');
+
+            // create a new event handler that will parse the repeatable values to the hidden inputs
+            // so they can be submited along with form when requesting some ajax endpoint
+            // we check that the event is not registered twice
+            if (typeof form_events === 'undefined' || !('backpack_field' in form_events) || !Object.values(form_events.backpack_field).find(e => e.namespace === 'parse_value')) {
+
+                form.on('backpack_field.parse_value', function(evt, element) {
+                    // TODO: element is the input that made the ajax request
```

Comment:
@prodixx here, we need to pass this `element` row, so in the controller that receives the ajax request we know what constrain to apply to show the options.

Reply:
@pxpm, I think you meant to mention @promatik? :)

Reply:
Hello @o15a3d4l11s2 

Hope you are doing well. 
Actually this was intended to @prodixx . I was having a conversation with him on our gitter channel :)

You should join us there, not every day, but sometimes we have good chats there, and some improvements in backpack comes from user feedback there :)

Anyway 10 points to you from awarenesses. 👍 

Have a nice weekend,
Pedro

Reply:
Ok soo... I've still got a problem with the name `backpack_field.parse_value`, sorry 😅👀😔

I propose we rename it to `backpack_field.calculate`. That way, we leave ourselves room to have more fields use it, as part of an entire cycle of supported events on fields:
- `backpack_field.init` / `backpack_field.initialize`, then `backpack_field.initalized`
- `backpack_field.calculate`, then `backpack_field.calculated`
- `backpack_field.hide`, then `backpack_field.hidden`
- `backpack_field.show`, then `backpack_field.shown`
- `backpack_field.delete`, then `backpack_field.deleted`

I wonder if we're going to remember this though... 😅 Or be able to find it here.

Alternatively... `backpack_field.compute`? Sorry for being a bitch about this @pxpm ... naming is hard... And I don't want us to have to create a breaking change for this in the next version, just because of the name.